src/OWML.ModHelper.Menus/ModMenu.cs

```diff
@@ -237,12 +237,10 @@ public object GetInputValue(string key)
 			{
 				return numberInput.Value;
 			}
-			var separator = GetSeparator(key);
-			if (separator != null)
+			if (GetSeparator(key) == null)
```

Comment:
Should Separator even be considered as input if null is returned regardless? Maybe just completely remove that check

Reply:
The issue is that it will create a log message without the check.